endoreg_db/management/commands/storage_management.py

```
@@ -0,0 +1,544 @@
+                                video.processed_file = None
+                                video.save(update_fields=['processed_file'])
+                                
+                            total_freed += file_size
+                            self.stdout.write(f"  Removed processed video {video.uuid}: {file_size / (1024**2):.1f} MB")
+                            
+                except Exception as e:
+                    logger.warning(f"Failed to clean processed video {video.uuid}: {e}")
+                    continue
+                    
+        except Exception as e:
+            logger.error(f"Failed to query processed videos: {e}")
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)